services/ui-src/src/components/export/ExportedOverlayModalReportSection.tsx

```diff
@@ -32,12 +32,7 @@ export const ExportedOverlayModalReportSection = ({
     }
   );
   return (
-    <Box
-      as="td"
-      mt="2rem"
-      data-testid="exportedOverlayModalPage"
-      sx={sx.container}
-    >
+    <Box mt="2rem" data-testid="exportedOverlayModalPage" sx={sx.container}>
```

Comment:
https://www.figma.com/file/oOJMUZerbafZefLQQ33GwI/MFP_Playground?type=design&node-id=2413-99200&mode=design&t=NjwIz8mBkNnfCMfF-0

This is the mockup for the export